webapp/blog/views.py

```diff
@@ -78,9 +86,8 @@ def snap_posts(snap):
                 except RequestException:
                     featured_media = None
 
-                transformed_article = logic.transform_article(
-                    article, featured_image=featured_media, author=None
-                )
+                transformed_article = helpers.transform_article(article)
+                transformed_article["image"] = featured_media
```

Comment:
@nottrobin What do you think about this? `feature_media` in this case is a string containing an HTML img tag (using Cloudinary). But this new version returns a dict object for the image.

Reply:
It would be nice if you'd stick to the structure of the blog module... Unless I misunderstood.

Reply:
Sorry, I made some modifications it is way better now.